mbuild/tests/test_lammpsdata.py

```diff
@@ -1,3 +1,5 @@
+from pathlib import Path
```

Comment:
Can we include a test case to ensure that all charges are correctly converted to LJ units? I do not see a test case for charges (correct me if I'm wrong).

P.S. All test cases are passing - ran them locally :)

Reply:
Done in commit dc75f9b!